src/gui/src/staDescriptors.cpp

```diff
@@ -204,7 +204,8 @@ Descriptor::Properties LibertyLibraryDescriptor::getProperties(
 
   SelectionSet corners;
   for (auto* corner : *sta_->corners()) {
-    for (const auto min_max : {sta::MinMax::min(), sta::MinMax::max()}) {
+    for (const sta::MinMax* min_max :
+         {sta::MinMax::min(), sta::MinMax::max()}) {
```

Comment:
What was the issue here?

Reply:
auto vs. auto&, I just made it explicit that these are pointers.

Reply:
Weird I don't see why you need a reference to the pointer.